js/components/ExampleChart.vue

```diff
@@ -1,6 +1,11 @@
 <template>
   <figure class="pull-right float-right vizr-example">
-    <img :src="imgSrc" :alt="messages.imgTitle" class="img-thumbnail" :title="messages.imgTitle">
+    <img
+      :src="imgSrc"
+      :alt="messages.imgTitle"
+      class="img-thumbnail"
+      :title="messages.imgTitle"
+    />
```

Comment:
Should the attributes be single-quoted for consistency?

Reply:
I don't know what the standard is in the Vue community, but the template linter introduced in Ember 3 insists on double quotes in templates. Consistency across template technologies would probably be good.